channel/src/main/java/com/synopsys/integration/alert/channel/azure/boards/distribution/search/AzureBoardsSearcher.java

```diff
@@ -93,7 +93,7 @@ public AzureBoardsSearcher(Gson gson, String organizationName, AzureBoardsIssueT
         Optional<IssuePolicyDetails> policyDetails = projectIssueModel.getPolicyDetails();
         Optional<String> optionalPolicyName = policyDetails.map(IssuePolicyDetails::getName);
         if (optionalPolicyName.isPresent()) {
-            categoryKey = AzureBoardsAlertIssuePropertiesManager.POLICY_ADDITIONAL_KEY_COMPATIBILITY_LABEL;
+            categoryKey = AzureBoardsAlertIssuePropertiesManager.CATEGORY_TYPE_POLICY_COMPATIBILITY_LABEL;
```

Comment:
Fixes inconsistent values in hidden custom-fields. This was causing querying issue where existing issues could not be found.